gradle/libs.versions.toml

```diff
@@ -27,8 +25,8 @@ activityCompose = "1.9.0"
 composeBom = "2024.04.00"
 mapsCompose = "4.3.3"
 materialIconsExtended = "1.6.5"
-mockitoAndroid = "2.12.0"
-mockitoCore = "3.11.2"
+mockitoAndroid = "5.12.0"
+mockitoCore = "5.12.0"
```

Comment:
A bit audacious to go two versions upward, but it passes the CI so I guess it's ok